app/views/publishers/invitations/jobs.html.slim

```diff
@@ -0,0 +1,14 @@
+- content_for :page_title_prefix, "Invite to apply for a job"
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("buttons.back"), href: back_url
+
+= form_for @step, url: { action: :update }, as: :invite do |f|
+  = f.govuk_error_summary
+
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      = f.govuk_collection_check_boxes(:job_ids, f.object.job_options, :value, :label, :hint,
+        legend: { size: "l", text: "Jobs to invite the candidate to apply for" },
```

Comment:
I think you can provide the text for this legend in `config/locales/forms.yml`. If you follow the patterns for other forms I guess it could be `t("helpers.legend.publishers_invite_form")`. Something like that anyway... lol